src/ui.c

```diff
@@ -233,11 +233,13 @@ static void attach_config_colors_to_screen(Config *config)
     char *css;
     int css_string_length = asprintf(&css,
         "* {\n"
-            "font: %s %s;\n"
```

Comment:
I was noticing pongo deprecation warnings in the seat0.log, so figured now would be a good time to change this. Easy to revert if it's unwanted